src/components/RegistrationSuccessPage/RegistrationSuccessPage.app.js

```diff
@@ -0,0 +1,21 @@
+// App.js
+
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import RegistrationSuccessPage from './RegistrationSuccessPage';
+import { AppRouterCacheProvider } from "@mui/material-nextjs/v13-appRouter";
+
+function App() {
+  return (
+    <Router>
+      <div>
+        <Switch>
+          {/* Other routes */}
+          <Route path="/registration/success" component={RegistrationSuccessPage} />
```

Comment:
This route should be create using folder routing system described in next.js docs.